backend/core/src/auth/entities/user.entity.ts

```diff
@@ -77,6 +86,11 @@ export class User {
   @Type(() => Date)
   dob?: Date | null
 
+  @Column("varchar", { nullable: true })
+  @Expose()
+  @IsPhoneNumber(null)
```

Comment:
This validator is missing groups parameter, please see other validators.

Reply:
Fixed.